Makefile.dep

```diff
@@ -48,6 +48,11 @@ ifneq (,$(filter sixlowpan,$(USEMODULE)))
   USEMODULE += vtimer
 endif
 
+ifneq (,$(filter ng_sixlowpan,$(USEMODULE)))
+  USEMODULE += ng_sixlowpan_netif
+  USEMODULE += ng_netbase
+endif
```

Comment:
Conceptionally, yes. To work: no. If we ever want to introduce stand-alone tests for every layer we would not need IPv6 here.